src/main/java/com/teamfive/disscard/DissCardController.java

```diff
@@ -14,98 +15,101 @@
 import java.util.List;
 
 /**
- * Controller class handling web ui and endpoints for the DissCard application
+ * Controller class handling web UI and endpoints for the DissCard application.
  * <h2>Front End</h2>
  * <p>
- *     Contains endpoints to generate dynamic HTML pages using Thymeleaf
+ *     Contains endpoints to generate dynamic HTML pages using Thymeleaf.
  * </p>
- *
  * <h2>Back End</h2>
  * <p>
  *     Contains endpoints that handle CRUD operations to the database through
- *     the card service
+ *     the card service.
  * </p>
- * @author Danny Murray
  */
 @Controller
 public class DissCardController {
 
-    // ===== Fields =====
+    private static final Logger log = LoggerFactory.getLogger(DissCardController.class);
 
-    @Autowired
-    ICardService cardService;
+    private final ICardService cardService;
 
-    Logger log = LoggerFactory.getLogger(this.getClass());
+    @Autowired
+    public DissCardController(ICardService cardService) {
+        this.cardService = cardService;
+    }
 
     // ===== Front-end endpoints =====
 
     /**
-     * Endpoint that generates the home page of the DissCard application's web UI
-     * @param model Object used to pass data to the thymeleaf html template
-     * @return DissCard home page
+     * Endpoint that generates the home page of the DissCard application's web UI.
+     * @param model Object used to pass data to the Thymeleaf HTML template.
+     * @return DissCard home page.
      */
-    @RequestMapping("/")
+    @GetMapping("/")
     public String index(Model model) {
         // Add data to model for use in view here
-
-        // Return name of html template to
         return "home";
     }
 
     // ===== Back-end endpoints =====
 
     /**
-     * Fetches a list of all cards registered with DissCard
-     * @return A list of JSON objects representing cards
+     * Fetches a list of all cards registered with DissCard.
+     * @return A list of JSON objects representing cards.
      */
     @GetMapping("/card/list")
     @ResponseBody
     public List<Card> fetchAllCards() {
+        log.info("Fetching all cards.");
         return cardService.getAll();
     }
 
     /**
-     * Fetches a card's info based on its ID
-     * @param id id of a card registered with DissCard
-     * @return A JSON object representing a card
+     * Fetches a card's info based on its ID.
+     * @param id ID of a card registered with DissCard.
+     * @return A ResponseEntity containing a JSON object representing a card.
      */
     @GetMapping("/card/info/{id}")
     public ResponseEntity<Card> fetchCardById(@PathVariable("id") int id) {
+        log.info("Fetching card with ID: {}", id);
```

Comment:
Standardization: The return statement now uses the buildResponse method, ensuring that the response creation logic is consistent across different endpoints. This helps maintain a standardized response format.